### DISCUSSION TITLE
Отправка электронных писем

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#342] [Как это сделать?] [answered]
Возможно настроить отправку электронных писем от имени пользователя username@yandex.ru, так чтобы эти письма находились также у него в папке отправленные? Сейчас письма доходят получателю только в папку входящие с отметкой "Письмо содержит неверную или поддельную информацию об отправителе. Также кто-то мог изменить текст письма после его отправки.", но этих писем нет в отправленных у отправителя.

**Comment by erpic-ru** [comment, marked as answer]
1. Нужно настроить[ кастомный SMTP](https://ru.docs.totum.online/functions#fn-emailSend-settings)
2. Чтобы иметь копию отправленного письма - можно дополнительно пересылать его из тотума на свой ящик и фильтрами внутри направлять в нужную папку

**Reply by vldekov** [reply]
В итоге каждый пользователь будет видеть свои письма в отправленных? или отправленные письма будут видны только в одной учетки от имени которой будет идти отправка?

**Comment by ttm-alexeypolunin** [comment]
Настроить отправку через smtp-яндекса. Насчет сохранения во входящих не знаю - смотрите их документацию по почте.

топик: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/136

в документации: https://ru.docs.totum.online/functions#fn-emailSend

**Reply by vldekov** [reply]
В итоге каждый пользователь будет видеть свои письма в отправленных? или отправленные письма будут видны только в одной учетки от имени которой будет идти отправка?

**Comment by vldekov** [comment]
В итоге каждый пользователь будет видеть свои письма в отправленных? или отправленные письма будут видны только в одной учетки от имени которой будет идти отправка?

**Reply by ttm-alexeypolunin** [reply]
Тотум не рассчитан на использование как почтового клиента.  Вы не можете от Тотума слать письма с 10 разных учеток. Тотум настраивается на одну учетку. Сохраняются ли исходящие при подключении через SMTP на почтовом сервере — зависит от настроек почтового сервера.  

Поэтому сделать так, что бы тотум слал при разных условиях письма от разных аккаунтов у вас врядли выйдет. Точнее давайте с оговорками — у тех, кто являетесь скилловым администратором собственного почтовика — может получится.

На яндексе у вас SMTP яндекса заблочит подстановку произвольного адреса, если он не будет совпадать с передаваемой учеткой.

**Что бы не валилось в спам и не считалось фишингом:**

— что бы почта имела хоть какое-то доверие — должны быть SPF/DKIM для домена (из значения показывает скрипт установки при завершении установки)
— что бы доверие было чуть-чуть выше — прочие DNS-записи типа MX, правильного rDNS итд (их показывает обычно почтовый сервер типа mailcow при добавлении домена), также вам их покажет яндекс если вы будете свой домен обслуживать у них
— что бы совсем не валилась в спам — должен быть заголовок list-unsubscribe и прочие мелочи, которые показывают сервисы «светит ли письму папка входящие»

* Когда вы отправляете с домена yandex.ru у них это все сделано и их сервер еще в трастовых адресах у других почтовиков

**Что вы можете использовать из простых решений:**

+ отправлять от одного технического ящика через smtp
+ использовать параметры в emailSend
    + replyto — при нажатии ответить ответ уйдет на указанный в этом параметре
    + hiddencopy — на этот адрес будет отправлена копия, которая не будет отображаться в адресатах основного письма
